ftw/noticeboard/tests/test_add_content.py

```diff
@@ -42,10 +43,37 @@ def test_add_notice(self, browser):
 
         browser.login().visit(category)
         factoriesmenu.add('Notice')
-        browser.fill({'Title': u'This is a Notice', 'Terms and Conditions': True})
+        browser.fill(
+            {
+                'Title': u'This is a Notice',
+                'Price': '100',
+                'Terms and Conditions': True,
+                'E-Mail': u'hans@peter.example',
+                'Text': u'Anything',
+            }
+        )
         browser.find_button_by_label('Save').click()
         self.assertEquals(u'This is a Notice', plone.first_heading())
 
+    @browsing
+    def test_notice_default_value_email_field(self, browser):
+        noticeboard = create(Builder('noticeboard').titled(u'Noticeboard'))
+        category = create(Builder('noticecategory')
+                          .having(conditions=RichTextValue('Something'))
+                          .titled(u'Category')
+                          .within(noticeboard))
+
+        user = create(Builder('user').with_roles('Site Administrator'))
```

Comment:
Does such a user have an email address by default? Otherwise, you'd be testing `u'' == u''`. You could assert that the email is not empty to show that explicitly.

Reply:
The user builder https://github.com/4teamwork/ftw.builder#users-builder always creates a E-Mail Address. Since I don't set any other params, the email address ends up to be random (I cannot assert it explicitly), but there is one for sure. I could set an explicit E-mail address, but IMHO i can relay on `ftw.builder`

Reply:
You could just check if there is an email. But if builder does that for you, it probably can stay that way. There is only a small risk that this can change.

Reply:
@busykoala 
That's fair, I added an assertion!